### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by thomasgouveia**
This PR contains the following improvements : 

- A more detailed README (motivation, status)... 
- Add a CODEOWNERS file to easily mention the @rik-org/core team on PR

### CODE REVIEW

**Review by mflagey — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
+## Conferences
+
+The project has been presented at the following events :
+
+- **Kubernetes Community Days 2023** | Paris, France | [Thomas Gouveia](https://github.com/thomasgouveia) and [Hugo Amalric](https://github.com/hugoamalric)
+- **[DevOps DDay 2022](https://www.youtube.com/watch?v=PS5aUSBdF-I)** | Marseille, France | [Thomas Gouveia](https://github.com/thomasgouveia) and [Hugo Amalric](https://github.com/hugoamalric)
+
+## License
+
+RIK is [MIT licensed](./LICENSE).
```
Comment (hugoamalric) [inline README.md:35]: I think you should add the copyright in the LICENSE file.
You can put `Polytech Montpellier` as the owner.

See https://github.com/rik-org/rik/blob/main/LICENSE#L189

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
\ No newline at end of file
+This section is under construction.
+
+## Troubleshooting
+
+**`cargo build` fails because cannot build `openssl-sys`**
+
+This is due to missing packages in your system, install `libssl-dev` to fix this.
+
+- Ubuntu: `sudo apt update && sudo apt install libssl-dev protobuf-compiler`
```
Comment (hugoamalric) [inline CONTRIBUTING.md]: Will be modified in an other PR ?

File: README.md (hunk 1 above)
Comment (hugoamalric) [reply README.md:35]: Also you can add, a link to the vercel documentation ?

**Review by hugoamalric — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (thomasgouveia) [reply CONTRIBUTING.md]: Yeah

File: README.md [hunk 3]
```diff
@@ -1,23 +1,35 @@
...
+Cloud adoption is growing every year. A lot of technologies were created to solve issues it brings on. One of the most popular technology when dealing with cloud is **containers**. As stated by the [CNCF annual survey of 2022](https://www.cncf.io/reports/cncf-annual-survey-2022/), containers are the new normal. A lot of companies are using containers in production to run their applications because of the portability, scalability and other benefits they offer. But in a world where elasticity is the key for building cost-efficient resilient applications, some tools must be used to automate management of the containers. This is where orchestrators like [Kubernetes](https://github.com/kubernetes/kubernetes), [HashiCorp Nomad](https://github.com/hashicorp/nomad) or others comes into play. 
 
-**`cargo build` fails because cannot build `openssl-sys`**
+The goal of RIK is not to replace Kubernetes or similar. It is a project mainly for educational purposes, and we want to provide a lightweight and simple code base to **help people understand underlying concepts of orchestrators**, or simply **to learn Rust** on a project related to a day-to-day issue: **orchestrate cloud applications**.
```
Comment (MaloPolese) [inline README.md:14]: To replace ? :eyes:  I don't think that rik purpose is to replace K8s.
More mimic k8s i think

File: CONTRIBUTING.md (hunk 2 above)
Comment (hugoamalric) [reply CONTRIBUTING.md]: Can you create an issue for it pls ? @thomasgouveia

File: README.md (hunk 3 above)
Comment (thomasgouveia) [reply README.md:14]: What do you mean? 

I want to say that we don't want to replace it, but create something similar that is easier to understand for people beginning in the domain.

Comment (MaloPolese) [reply README.md:14]: ah my bad i didn't read the "not"

Comment (MaloPolese) [reply README.md:14]: LGTM

**Review by MaloPolese — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (thomasgouveia) [reply CONTRIBUTING.md]: Done here https://github.com/rik-org/rik/issues/40

**Review by sameo — CHANGES_REQUESTED** [review]
You're also relicensing the project from Apache2 to MIT:

* Why?
* You *must* get approvals from all copyright holders before doing so
* This should be a separate PR of its own.

**Comment by thomasgouveia** [comment]
Hi @sameo,

I will remove the license commit. For sure, the approval of all contributors is required. I put a MIT because we usually use it in all of our other projects. We can discuss about it if you want !

**Review by hugoamalric — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,23 +1,35 @@
...
+## Contributing
+
+RIK is open-source and contributions are highly welcome. You will find all the guidance to contribute to the project in the [CONTRIBUTING.md](./CONTRIBUTING.md) file.
+
+## Conferences
+
+The project has been presented at the following events :
+
+- **Kubernetes Community Days 2023** | Paris, France | [Thomas Gouveia](https://github.com/thomasgouveia) and [Hugo Amalric](https://github.com/hugoamalric)
+- **[DevOps DDay 2022](https://www.youtube.com/watch?v=PS5aUSBdF-I)** | Marseille, France | [Thomas Gouveia](https://github.com/thomasgouveia) and [Hugo Amalric](https://github.com/hugoamalric)
```
Comment (AlexandreBrg) [inline README.md:31]: As both refer to the same conference, please keep only your preferred meetup.

**Review by AlexandreBrg — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (AlexandreBrg) [reply README.md:35]: Depending on the choice we do

[the remark this answers is not in the document]
Comment (thomasgouveia) [reply README.md:35]: Yes I forgot to update, I'll do it

File: README.md [hunk 5]
```diff
@@ -1,23 +1,34 @@
...
 
-<div>
-<img src="https://img.shields.io/badge/Rik-rust-orange?style=for-the-badge&logo=appveyor" />
-<img src="https://img.shields.io/github/workflow/status/thomasgouveia/rik/RIK%20CI?style=for-the-badge" />
-<img alt="Discord" src="https://img.shields.io/discord/863020591984148502?style=for-the-badge">
-</div>
+## What is RIK ?
 
-### Contributing
+RIK (pronounced /rɪk/) is an experimental workload orchestrator written in Rust that aims to be able to natively schedule **containers** and **virtual machines** workloads. RIK stands for **R**ust **i**n **K**ubernetes.
```
[the remark this answers is not in the document]
Comment (thomasgouveia) [reply README.md:8]: We are working actually on Morty, our FaaS platform that use RIK under the hood to schedule microVM to run functions. By natively, I would like to say that we don't need to install anything else to be able to run certain type of workloads

File: README.md [hunk 6]
```diff
@@ -1,23 +1,34 @@
...
+
+## Contributing
+
+RIK is open-source and contributions are highly welcome. You will find all the guidance to contribute to the project in the [CONTRIBUTING.md](./CONTRIBUTING.md) file.
+
+## Conferences
+
+The project has been presented at the following events :
+
+- **Kubernetes Community Days 2023** | Paris, France | [Thomas Gouveia](https://github.com/thomasgouveia) and [Hugo Amalric](https://github.com/hugoamalric)
```
[the remark this answers is not in the document]
Comment (thomasgouveia) [reply README.md:30]: I added it

**Review by sameo — APPROVED** [review]

**Pull request merged**